src/rest.js

```diff
@@ -34,7 +34,7 @@ export class Rest {
   request(method, path, body, options = {}) {
     let requestOptions = extend(true, {headers: {}}, this.defaults, options, {method, body});
 
-    let contentType = requestOptions.headers['Content-Type'];
+    let contentType = requestOptions.headers['Content-Type'] || requestOptions.headers['content-type'];
```

Comment:
Perhaps it's a nice idea to capture this while setting and toLowerCase() the whole thing. This doesn't solve `Content-type` for instance (which I've also seen).

Either way, these are the most common ones so I approve 👍